src/redux/categories/categories.js

```diff
@@ -0,0 +1,12 @@
+const CHECKED = 'app/reducer/CHECKED';
+
+const checkReducer = (state = [], action) => {
+  switch (action.type) {
+    case CHECKED:
+      return 'Under construction';
+    default: return state;
+  }
+};
```

Comment:
- [ ] Also when I click on the `checkstatus` button nothing happens, please kindly it as well 😉